testsuite/MDAnalysisTests/converters/test_openmm.py

```diff
@@ -65,18 +65,19 @@ def omm_sim_uni(self):
         return mda.Universe(simulation)
 
     def test_dimensions(self, omm_sim_uni):
-        assert_almost_equal(
+        assert_allclose(
             omm_sim_uni.trajectory.ts.dimensions,
             np.array([20., 20., 20., 90., 90., 90.]),
-            3,
-            "OpenMMBasicSimulationReader failed to get unitcell dimensions " +
-            "from OpenMM Simulation Object",
+            rtol=0,
+            atol=1e-3,
+            err_msg="OpenMMBasicSimulationReader failed to get unitcell " +
+                    "dimensions from OpenMM Simulation Object",
```

Comment:
Can you use implicit continuation here? i.e.

```suggestion
            err_msg=("OpenMMBasicSimulationReader failed to get unitcell "
                     "dimensions from OpenMM Simulation Object"),
```